domains/transfer/Shared/DataContext/DataContext.csproj

```diff
@@ -8,14 +8,14 @@
 
   <ItemGroup>
     <PackageReference Include="Audit.EntityFramework.Core" Version="21.0.1" />
-    <PackageReference Include="Microsoft.EntityFrameworkCore" Version="7.0.11" />
-    <PackageReference Include="Microsoft.EntityFrameworkCore.Design" Version="7.0.11">
+    <PackageReference Include="Microsoft.EntityFrameworkCore" Version="8.0.0" />
+    <PackageReference Include="Microsoft.EntityFrameworkCore.Design" Version="8.0.0">
       <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
     </PackageReference>
-    <PackageReference Include="Microsoft.EntityFrameworkCore.Tools" Version="7.0.11">
+    <PackageReference Include="Microsoft.EntityFrameworkCore.Tools" Version="8.0.0">
       <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
     </PackageReference>
-    <PackageReference Include="Npgsql.EntityFrameworkCore.PostgreSQL" Version="7.0.11" />
+    <PackageReference Include="Npgsql.EntityFrameworkCore.PostgreSQL" Version="8.0.0" />

```

Comment:
There is an ongoing PR, relating to upgrading NuGet packages:

https://github.com/Energinet-DataHub/energy-origin/pull/561

Reply:
I wasn't able to migrate the database without updating the nuget packages. We are currently in a state where migrations are failing. So me and Martin can't get started with the project.